src/sagemaker/tensorflow/estimator.py

```diff
@@ -171,6 +171,7 @@ class TensorFlow(Framework):
     """Handle end-to-end training and deployment of user-provided TensorFlow code."""
 
     __framework_name__ = 'tensorflow'
+    LATEST_VERSION = '1.12'
```

Comment:
The default is defined in the defaults.py: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/defaults.py

Reply:
I am trying to be consistent with other framework here. it's defined in the class for pytorch.

Reply:
and mxnet and chainer

Reply:
This is what we agreed on a few month ago to prevent silently switching to the container with the latest framework version installed whenever it's released since frameworks have breaking changes.

We have been switching to this scheme slowly as new framework versions get released and TF seems to be the last one :)